flake.nix

```diff
@@ -49,7 +49,7 @@
           in
             l.attrValues {
               inherit serapi;
-              inherit (ocamlPackages) yojson cmdliner uri;
+              inherit (ocamlPackages) yojson cmdliner uri dune-build-info;
```

Comment:
I should note that this is really `dune-build-info.2.6.0`. I am pretty sure that it hasn't changed in 3.6 and we should be safe under the `V1` API anyway. It doesn't appear to cause any ill-effects in the Nix builds.